app/src/main/java/com/tananaev/passportreader/MainActivity.java

```diff
@@ -371,6 +408,18 @@ protected Exception doInBackground(Void... params) {
             return null;
         }
 
+        private String bitmapToBase64(Bitmap bitmap) {
+            ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+            bitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayOutputStream);
```

Comment:
Do we really need to re-compress image? Ideally we want to send original.

Reply:
yes we definitely want to preserve the quality of the image.  i think the name of this method could a little misleading though, i think what it is essentially doing is encoding the bitmap to the specified format, so provided we use a lossless format (now PNG, in latest commit), image quality should be preserved ok.  Using .compress appears to be the accepted way to convert an Android Bitmap to a base64 string, at least, I'm not aware of another way.

Reply:
Yes, but we can't we just use original image?

Reply:
ahh, yes I see what you mean now, i'm done that on a new commit.  i can squash these commits if you like btw.